src/node/README.md

```diff
@@ -1,7 +1,7 @@
 
-# Node.js (via nvm) and yarn (node)
+# Node.js (via nvm), yarn and pnpm (node)
```

Comment:
Can we revert these changes to avoid merge conflicts?

Reply:
Reverted.